### DISCUSSION TITLE
Unable to redirect search results to a custom search results page with a PnP search web part configured

### DISCUSSION THREAD

**Discussion by DaveMT65** [microsoft-search/pnp-modern-search#1644] [Q&A]
We are not able to redirect search results to a custom search results page with a PnP search web part configured. We have a modern communication site on a GCC High tenant. The same feature works fine on the commercial tenant.

**Comment by wobba** [comment]
Redirecting a search box is outside this web part project, and the header search box is not shipped in GCC-H afaik.
